### PR TITLE
Update daffodil-site as vscode extension 1.0.0 was released.

### PR DESCRIPTION
**Pull request by Shanedell**

### CODE REVIEW

**Review by tuxji — APPROVED** [review]
+1

I know there's no previous daffodil-vscode release which needs its distribution links updated, but should the daffodil-vscode directory have a symbolic link called "latest" pointing to the latest release of daffodil-vscode?

**Comment by Shanedell** [comment]
I am not sure if that would be necessary since we don't do it for Daffodil. However, once we start also publishing to the VS Code Marketplace then the latest is always installed, usually you will need to reload VS Code for it to update, but then if you want a older version of the extension you will need to go to the downloads page and download the VSIX file.

File: site/_vscode/1.0.0.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 
-released: false
+released: true
 apache: true
-title: VS Code Extension 1.0.0
-date: 2022-03-17
+title: 1.0.0
```
Comment (stevedlawrence) [inline site/_vscode/1.0.0.md:5]: This needs to include "VS Code Extension", otherweise the title is just 'Apache Daffodil | 1.0.0" and there's no difference between a VSCode release title and a normal Daffodil release title.

**Comment by stevedlawrence** [comment]
> I know there's no previous daffodil-vscode release which needs its distribution links updated, but should the daffodil-vscode directory have a symbolic link called "latest" pointing to the latest release of daffodil-vscode?

The `latest` symlink that daffodil uses is just for documentation:

https://daffodil.apache.org/docs/latest/javadoc/

We don't currently do the same thing for releases.

File: site/_vscode/1.0.0.md (hunk 1 above)
Comment (stevedlawrence) [reply site/_vscode/1.0.0.md]: Which box?

**Comment by Shanedell** [comment]
@stevedlawrence Is this good to merge I will send out the announcement after I merge this in

**Comment by stevedlawrence** [comment]
Ah, I now see why you removed the "VS Code Extension", since we use the `title` in places where we want to show just the number. But we do want `VS Code Extension` in some places...

What if you switch back to just the version number in the title like you had and then we have this change:
```diff
diff --git a/site/_includes/themes/apache/_navigation.html b/site/_includes/themes/apache/_navigation.html
index 1205789..39050f9 100644
--- a/site/_includes/themes/apache/_navigation.html
+++ b/site/_includes/themes/apache/_navigation.html
@@ -63,7 +63,7 @@
 {% else %}
 <div class="title">
   <div class="container">
-    <h2>{% if page.collection == "releases" %}{% if page.apache %}Apache {% endif %}Daffodil {% endif %}{{ page.title }}</h2>
+    <h2>{% if page.collection == "releases" %}{% if page.apache %}Apache {% endif %}Daffodil {% endif %}{% if page.collection == "vscode" %}VS Code Extension {% endif %}{{ page.title }}</h2>
   </div>
 </div>
 {% endif %}
diff --git a/site/_includes/themes/apache/default.html b/site/_includes/themes/apache/default.html
index 3adc99a..9f68b90 100644
--- a/site/_includes/themes/apache/default.html
+++ b/site/_includes/themes/apache/default.html
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Apache Daffodil | {{ page.title }}</title>
+    <title>Apache Daffodil | {% if page.collection == "vscode" %}VS Code Extension {% endif %}${{ page.title }}</title>
     {% if page.description %}<meta name="description" content="{{ page.description }}">{% endif %}
     <meta name="author" content="{{ site.author.name }}">
```

Not tested, but the idea is that the `title` variable is just the version number like you had before, and the places where we display that version number (the <title> tag and the page header) and want more than just the version we special case and add the `VS Code Extension` string.

It's a big hacky, but minimizes changes.

**Comment by stevedlawrence** [comment]
Minor, but found another place where VS Code Extension is needed:

```diff
diff --git a/site/_layouts/vscode.html b/site/_layouts/vscode.html
index d21f4d6..3cbe28c 100644
--- a/site/_layouts/vscode.html
+++ b/site/_layouts/vscode.html
@@ -5,9 +5,9 @@ permalink: /vscode/release-notes-:title
 
 {% unless page.released %}
   <div class="alert alert-warning">
-    Apache Daffodil {{ page.title }} has not yet been released! The artifacts
+    Apache Daffodil VS Code Extension {{ page.title }} has not yet been released! The artifacts
     and release notes below are drafts for a proposed release of Apache
-    Daffodil which has not yet occurred.
+    Daffodil VS Code Extension which has not yet occurred.
   </div>
 {% endunless %}
 

```

**Comment by Shanedell** [comment]
@stevedlawrence Both those changes mentioned above should be there now. If it looks good let me know I will squash before merging

**Comment by stevedlawrence** [comment]
+1 :+1:  looks good to squash and merge

**Comment by stevedlawrence** [comment]
Note sure how it happened, but it looks like I accidentally added a `$` too the title, I'll open a PR to fix it.

**Pull request merged**